modules/exploits/unix/webapp/drupal_drupalgeddon2.rb

```diff
@@ -0,0 +1,103 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Drupal Drupalgeddon 2',
+      'Description'    => %q{
+        This module exploits a vulnerability.
+      },
+      'Author'         => [
+        'Jasper Mattsson', # Vulnerability discovery
+        'a2u',             # Proof of concept
+        'Nixawk',          # Proof of concept
+        'wvu'              # Metasploit module
+      ],
+      'References'     => [
+        ['CVE', '2018-7600'],
+        ['URL', 'https://www.drupal.org/sa-core-2018-002'],
+        ['URL', 'https://greysec.net/showthread.php?tid=2912'],
+        ['URL', 'https://research.checkpoint.com/uncovering-drupalgeddon-2/'],
+        ['URL', 'https://github.com/a2u/CVE-2018-7600'],
+        ['URL', 'https://github.com/nixawk/labs/issues/19']
+      ],
+      'DisclosureDate' => 'Mar 28 2018',
+      'License'        => MSF_LICENSE,
+      'Platform'       => 'unix',
+      'Arch'           => ARCH_CMD,
+      'Privileged'     => false,
+      'Targets'        => [
+        ['Drupal < 7.58, < 8.3.9, < 8.4.6, < 8.5.1', {}]
```

Comment:
I understand that Drupal 6.x is also affected:

"This issue also affects Drupal 6. Drupal 6 is End of Life. For more information on Drupal 6 support please contact a D6LTS vendor."
Source: https://www.drupal.org/sa-core-2018-002